components/list/src/ListItem.js

```diff
@@ -0,0 +1,157 @@
+/* @jsx jsx */
+
+import React, { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, css, useTheme } from '@westpac/core';
+import { List } from './List';
+
+// ==============================
+// Utils
+// ==============================
+const getListIcon = ({ appearance, color, icon: Icon }) => {
+	const { list } = useTheme();
+
+	const style = {
+		position: 'absolute',
+		left: 0,
+		top: 2,
+	};
+
+	switch (appearance) {
+		case 'bullet':
+			return (
+				<svg
+					aria-labelledby="title-bullet-list"
+					version="1.1"
+					xmlns="http://www.w3.org/2000/svg"
+					width="16px"
+					height="16px"
+					viewBox="0 0 16 16"
+					css={style}
+				>
+					<circle
+						r="4"
+						cx="8"
+						cy="8"
+						stroke={`${list[appearance][color].color}`}
+						strokeWidth="1"
+						fill={`${list[appearance][color].color}`}
+					/>
```

Comment:
With `cx="8"` and `cy="8"` and a `strokeWidth="1"` it renders a 9px diameter circle... we need 8px

Reply:
Just looked into this... seems the stroke is placed inside or outside depending on the browser 😬